table_schema_cache.go

```diff
@@ -189,10 +190,7 @@ func LoadTables(db *sql.DB, tableFilter TableFilter, columnCompressionConfig Col
 			tableLog := dbLog.WithField("table", tableName)
 			tableLog.Debug("caching table schema")
 
-			// Sanity check
-			if len(tableSchema.PKColumns) != 1 {
-				err = fmt.Errorf("table %s has %d primary key columns and this is not supported", tableName, len(tableSchema.PKColumns))
-				logger.WithError(err).Error("invalid table")
+			if err := tableSchema.setPaginationKeyColumn(dbname, tableName, paginateByColumnConfig); err != nil {
 				return tableSchemaCache, err
 			}
 
```

Comment:
few lines down, we have the error message "table ... is using a non-numeric primary key column and this is not supported"

could you reword that?